app/views/admin/users/widgets/_credits.html.erb

```diff
@@ -23,20 +23,22 @@
       <div class="flex gap-2 items-center">
         <select name="" id="balance-type" class="crayons-select w-auto" required>
           <option value="add">Add</option>
-          <option value="subtract">Subtract</option>
+          <% if current_credits.positive? %>
+            <option value="subtract">Subtract</option>
+          <% end %>
         </select>
 
-        <input type="number" inputmode="numeric" placeholder="10" size="5" min="1" max="9999" required class="crayons-textfield w-auto">
+        <input type="number" inputmode="numeric" size="5" min="1" max="9999" value="1" required class="crayons-textfield w-auto">
 
         <span class="color-secondary flex-1">credits</span>
       </div>
     </div>
     <div class="crayons-field">
       <label for="balance-note" class="crayons-field__label">Why are you adjusting credits?</label>
-      <textarea name="" id="balance-note" class="crayons-textfield" placeholder="Explanation..."></textarea>
+      <textarea name="" id="balance-note" class="crayons-textfield" required></textarea>
```

Comment:
The label implies what the placeholder is saying and again, there's a colour contrast issue.

Reply:
Color contrast issue is apparently an issue on a component level. Not related to this PR. Will be fixing that separately.